analyzer/src/main/java/nl/tudelft/mavensecrets/config/Config.java

```diff
@@ -45,6 +45,19 @@ public interface Config {
      */
     File getLocalRepository();
 
+    /**
+     * Get the seed for the random sampling.
+     * @return The seed.
+     */
+    long getSeed();
+
+
+    /**
+     * Get the percentage of the dataset to be sampled.
+     * @return The sample percentage.
+     */
+    float getSamplePercent();
```

Comment:
There is no reason to use floats over doubles, but it does mean the value is less precise, which is an unnecessary downside.

Reply:
Hater